packages/paste-core/components/chat-log/__tests__/customization.spec.tsx

```diff
@@ -0,0 +1,221 @@
+import * as React from 'react';
+import {screen, render} from '@testing-library/react';
+import {CustomizationProvider} from '@twilio-paste/customization';
+import {ChatMessage, ChatBubble} from '../src';
+
+describe('Customization', () => {
+  it('should add custom styles to ChatMessage', () => {
+    render(
+      <CustomizationProvider
+        baseTheme="default"
+        theme={TestTheme}
+        elements={{CHAT_MESSAGE: {marginBottom: 'space100'}}}
+      >
+        <ChatMessage variant="inbound" data-testid="inbound-message">
+          test
+        </ChatMessage>
+      </CustomizationProvider>
+    );
+
+    const inboundMessage = screen.getByTestId('inbound-message');
+    expect(inboundMessage).toHaveStyleRule('margin-bottom', '2.25rem');
+  });
+
+  it('should set element data attribute on ChatMessage', () => {
+    render(
+      <CustomizationProvider
+        baseTheme="default"
+        theme={TestTheme}
+        elements={{CHAT_MESSAGE: {marginBottom: 'space100'}}}
+      >
+        <ChatMessage variant="inbound" data-testid="inbound-message">
+          test
+        </ChatMessage>
+      </CustomizationProvider>
+    );
+
+    const inboundMessage = screen.getByTestId('inbound-message');
+    expect(inboundMessage.getAttribute('data-paste-element')).toEqual('CHAT_MESSAGE');
+  });
+
+  it('should add custom styles to ChatMessage with a custom element data attribute', () => {
+    render(
+      <CustomizationProvider baseTheme="default" theme={TestTheme} elements={{foo: {marginBottom: 'space100'}}}>
+        <ChatMessage element="foo" variant="inbound" data-testid="inbound-message">
+          test
+        </ChatMessage>
+      </CustomizationProvider>
+    );
+
+    const inboundMessage = screen.getByTestId('inbound-message');
+    expect(inboundMessage).toHaveStyleRule('margin-bottom', '2.25rem');
+  });
+
+  it('should set custom element data attribute on ChatMessage', () => {
+    render(
+      <CustomizationProvider baseTheme="default" theme={TestTheme} elements={{foo: {marginBottom: 'space100'}}}>
+        <ChatMessage element="foo" variant="inbound" data-testid="inbound-message">
+          test
+        </ChatMessage>
+      </CustomizationProvider>
+    );
+    const inboundMessage = screen.getByTestId('inbound-message');
+    expect(inboundMessage.getAttribute('data-paste-element')).toEqual('foo');
+  });
```

Comment:
Suggestion: test coverage for chat message custom styles

I know it doesn't _do anything_ right now, but with your public API you are exposing the ability to add custom styles to the chat message (on top of the chat bubble), for both inbound and outbound variants, but we don't have tests covering that. Unlike chat bubble where we do.

(Unless I'm tired and can't see them)